### ISSUE TITLE
Predictive text behaves something odd

### ISSUE THREAD

**Issue by kpatnayakuni** [PowerShell/PSReadLine#1591]
PSReadLine is behaving something odd with the predictions, when the prediction is in multiple lines and if we are not continuing with the predicted command then it leaves the marks from the 2nd line in the screen, notice the same in the 2nd Pic and 1st pic shows the prediction.

![image](https://user-images.githubusercontent.com/31788728/83952692-07d1cd80-a858-11ea-89d4-2e284caee33a.png)

![image](https://user-images.githubusercontent.com/31788728/83952730-58e1c180-a858-11ea-960c-63858fbfdcea.png)

## Environment

```none
PS version: 7.0.1
PSReadline version: 2.1.0-beta2
os: 10.0.19041.1 (WinBuild.160101.0800)
PS file version: 7.0.1.0
HostName: ConsoleHost (Windows Terminal)
BufferWidth: 209
BufferHeight: 52
```

## Steps to reproduce

You can see in the pics that I have attached.

## Expected behavior

It should wipe the remaining predicted text
## Actual behavior
It leaves the predicted text from the 2nd line on the screen

**Comment by daxian-dbw** [comment]
This is a known issue. A better implementation is need to erase the suggestion text as needed.

**Comment by daxian-dbw** [comment]
@kpatnayakuni Please keep this open to track the issue.

**Comment by daxian-dbw** [comment]
The fix will be included in the 2.2.0-beta1 release.
